gensim/models/ldamodel.py

```diff
@@ -82,6 +82,12 @@
 
     >>> lda = LdaModel(common_corpus, num_topics=50, alpha='auto', eval_every=5)  # learn asymmetric alpha from data
 
+LDA returns complete topic probability distribution when you set minimum_probability to 0.0
```

Comment:
This is too vague, and IMO more confusing than illuminating.

What is "complete"?
Why would you set `minimum_probability` manually? Or indeed *not* set it and keep the default?

Note that Gensim only returns *sparse* bow tuples: no explicit zeros, no dense vectors.